src/sp_config.c

```diff
@@ -189,6 +189,10 @@ void sp_disabled_function_list_free(sp_node_t* list) {
       sp_disabled_function* df = cursor->data;
       if (df && df->functions_list)
           sp_list_free(df->functions_list);
+      if (df) {
+	free_sp_tree(df->param);
+	free_sp_tree(df->var);
+      }
```

Comment:
change : free_sp_tree to sp_tree_free to be coherent with `sp_list_*` (commit inc)